portal/src/locale-data/en.json

```diff
@@ -13,6 +14,10 @@
   "disable": "Disable",
   "confirm": "Confirm",
   "cancel": "Cancel",
+  "disconnect": "Disconnect",
+  "verified": "Verified",
```

Comment:
We have `verified` and `UserDetails.connected-identities.verified`, is there any difference in their semantics?

Reply:
it is duplicated, let's remove `UserDetails.connected-identities.verified` and rename `UserDetails.connected-identities.unverified` to `unverified`